packages/typespec-ts/test/modularUnit/scenarios/samples/operations/armCurdOperations.md

```diff
@@ -0,0 +1,248 @@
+      },
+      "location": "itajgxyqozseoygnl"
+    }
+  },
+  "responses": {
+    "200": {}
+  }
+}
+```
+
+```json for Employees_Delete
+{
+  "title": "Employees_Delete",
+  "operationId": "Employees_Delete",
+  "parameters": {
+    "api-version": "2021-10-01-preview",
+    "subscriptionId": "11809CA1-E126-4017-945E-AA795CD5C5A9",
+    "resourceGroupName": "rgopenapi",
+    "employeeName": "5vX--BxSu3ux48rI4O9OQ569"
+  },
+  "responses": {
+    "202": {}
+  }
+}
+```
+
+## Samples
+
+Generate samples for arm cases:
+
+```ts samples
+/** This file path is /samples-dev/operationsListSample.ts */
+import { ContosoClient } from "@azure/internal-test";
+import { DefaultAzureCredential } from "@azure/identity";
+
+/**
+ * This sample demonstrates how to list the operations for the provider
+ *
+ * @summary list the operations for the provider
+ * x-ms-original-file: 2021-10-01-preview/json_for_Operations_List.json
+ */
+async function operationsList() {
+  const credential = new DefaultAzureCredential();
+  const subscriptionId = "00000000-0000-0000-0000-00000000000";
+  const client = new ContosoClient(credential, subscriptionId);
+  const resArray = new Array();
+  for await (let item of client.operations.list()) {
+    resArray.push(item);
+  }
+
+  console.log(resArray);
+}
+
+async function main() {
+  operationsList();
+}
+
+main().catch(console.error);
+
+/** This file path is /samples-dev/employeesCreateOrUpdateSample.ts */
+import { ContosoClient } from "@azure/internal-test";
+import { DefaultAzureCredential } from "@azure/identity";
+
+/**
+ * This sample demonstrates how to create a Employee
+ *
+ * @summary create a Employee
+ * x-ms-original-file: 2021-10-01-preview/json_for_Employees_CreateOrUpdate.json
+ */
+async function employeesCreateOrUpdate() {
+  const credential = new DefaultAzureCredential();
+  const subscriptionId = "11809CA1-E126-4017-945E-AA795CD5C5A9";
+  const client = new ContosoClient(credential, subscriptionId);
+  const result = await client.employees.createOrUpdate(
+    "rgopenapi",
+    "9KF-f-8b",
+    {
+      properties: {
+        age: 30,
+        city: "gydhnntudughbmxlkyzrskcdkotrxn",
+        profile: "ms",
+      },
+      tags: { key2913: "urperxmkkhhkp" },
+      location: "itajgxyqozseoygnl",
+    },
+  );
+  console.log(result);
+}
+
+async function main() {
+  employeesCreateOrUpdate();
+}
+
+main().catch(console.error);
+
+/** This file path is /samples-dev/employeesDeleteSample.ts */
+import { ContosoClient } from "@azure/internal-test";
+import { DefaultAzureCredential } from "@azure/identity";
+
+/**
+ * This sample demonstrates how to delete a Employee
+ *
+ * @summary delete a Employee
+ * x-ms-original-file: 2021-10-01-preview/json_for_Employees_Delete.json
+ */
+async function employeesDelete() {
+  const credential = new DefaultAzureCredential();
+  const subscriptionId = "11809CA1-E126-4017-945E-AA795CD5C5A9";
+  const client = new ContosoClient(credential, subscriptionId);
```

Comment:
other than the subscriptionId, is there any other differences between mgmt plane sample and data plane sample?

Reply:
i don't think there are any other differences for now.

Reply:
i don't think we have others for now.